File: C++/프로그래머스/0/181921. 배열 만들기 2/배열 만들기 2.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

vector<int> solution(int l, int r) {
    vector<int> answer;
    
    string str; bool flag;
    while(l % 5 != 0) {
        l++;
    }
    for(long i = l; i <= r; i += 5) {
        flag = true;
        str = to_string(i);
        for(int j = 0; j < str.length(); j++) {
            if(str[j] != '5' && str[j] != '0') {
                flag = false;
                break;
            }
        }
        if(flag) {
            cout << i << " ";
            answer.push_back(i);
        }
    }
    
    if(answer.empty()) {
        answer.push_back(-1);
    }
    
    return answer;
}

File: Python3/프로그래머스/0/181921. 배열 만들기 2/배열 만들기 2.py
def solution(l, r):
    answer = []
    
    #l을 5의 배수로
    while l % 5 != 0: l+=1
    for i in range(l, r + 1, 5):
        if str(i).replace("5", "").replace("0", "") == "":
            answer += [i]
    
    if len(answer) == 0: answer = [-1]
    return answer